Assets/Scripts/Win.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class Win : MonoBehaviour
+{
+    public GameObject uiObject;
```

Comment:
If this is being set in the inspector, I think it is better to use private instead of public and use [SerializeField] to make it visible in the inspector. If you end up doing this, don't forget to make a tooltip for it as well.